ui/apps/platform/cypress/integration/systemHealth/certificateHealthCards.test.js

```diff
@@ -0,0 +1,150 @@
+import withAuth from '../../helpers/basicAuth';
+import {
+    credentialForCentralExpiryAlias,
+    credentialForCentralDbExpiryAlias,
+    credentialForScannerExpiryAlias,
+    setClock,
+    visitSystemHealth,
+} from '../../helpers/systemHealth';
+
+const statusSelectorCentral = 'div:has(.pf-v5-c-card__header:contains("Central certificate"))';
+const statusSelectorCentralDb =
+    'div:has(.pf-v5-c-card__header:contains("Central Database certificate"))';
+const statusSelectorScanner =
+    'div:has(.pf-v5-c-card__header:contains("StackRox Scanner certificate"))';
+
+describe('System Health Certificate Health Cards', () => {
+    withAuth();
+
+    describe('Central certificate', () => {
+        it('should have widget and up to date text', () => {
+            const currentDatetime = new Date('2025-05-21T02:04:59.377369440Z'); // about 355 days until expiry
+            const expiry = '2026-05-20T03:04:59.377369440Z';
+
+            const staticResponseMap = {
+                [credentialForCentralExpiryAlias]: { body: { expiry } },
+            };
+
+            setClock(currentDatetime); // call before visit
+            visitSystemHealth(staticResponseMap);
+
+            cy.get(`${statusSelectorCentral}:contains(expires in 12 months)`);
```

Comment:
As a habit, it is safer to enclose `contains` argument in double quotes: `contains("expires in 12 months")`